examples/example-pu/pom.xml

```diff
@@ -19,13 +19,18 @@
 		</dependency>
 		<dependency>
 			<groupId>${project.groupId}</groupId>
-			<artifactId>ymer-test</artifactId>
+			<artifactId>ymer-test-junit5</artifactId>
 			<scope>test</scope>
 			<version>${project.version}</version>
 		</dependency>
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-params</artifactId>
```

Comment:
I suggest just adding a test dependency to `junit-jupiter`, it includes `-api`, `-engine` and `-params`

Reply:
Well spotted! I didn't check the changes in the example app too closely. I'll fix that!